client/stylesheets/components/_shortlist.scss

```diff
@@ -0,0 +1,95 @@
+shortlist {
+  $base-spacing: 1.5em !default;
```

Comment:
Here `!default` make sense, as `$base-spacing` could already be defined and shared among components.